tests/test_loss.py

```diff
@@ -27,3 +27,35 @@ def test_pseudo_huber_loss():
 
 def test_flat_bottom_loss():
     _assert_basic_loss_properties(flat_bottom_loss)
+
+
+def _scalar_truncated_residual(prediction, label, reliable_interval):
+    lower, upper = reliable_interval
+    assert lower < upper
+
+    if (label >= lower) and (label <= upper):
+        residual = prediction - label
+    elif label < lower:
+        residual = max(0, prediction - lower)
+    elif label > upper:
+        residual = min(0, prediction - upper)
+    else:
+        raise (RuntimeError('unsatisfiable reliable_range'))
```

Comment:
When would this assertion be raised?

Reply:
sorry just realized this is in the test file! don't really care then

Reply:
Didn't have a specific error in mind, just being conservative -- this line might be unreachable, and I can remove it.

(Wanted to hear something in case we later pass some combination of nans, infs, +infs, and reals in `label`, `lower`, `upper` that doesn't get caught by the if statements.)